config/ext_jenkins-release_privatek8s.yaml

```diff
@@ -0,0 +1,692 @@
+      #                         a(class: 'jenkins.branch.BranchProperty-array') {
+      #                             'jenkins.branch.NoTriggerBranchProperty'()
+      #                         }
+      #                     }
+      #                 }
+      #             }
+      #           }
+      #         }
+      #     - script: >
+      #         multibranchPipelineJob('core/stable-rc/release') {
+      #           displayName "Release"
+      #           description "Jenkins Core Release"
+      #           branchSources {
+      #             github {
+      #               id('2020121601')
+      #               scanCredentialsId('github-access-token')
+      #               repoOwner('jenkins-infra')
+      #               repository('release')
+      #             }
+      #           }
+      #           factory {
+      #             workflowBranchProjectFactory {
+      #               scriptPath('Jenkinsfile.d/core/stable-rc')
+      #             }
+      #           }
+      #           configure {
+      #             it / sources / 'data' / 'jenkins.branch.BranchSource' << {
+      #                 strategy(class: 'jenkins.branch.DefaultBranchPropertyStrategy') {
+      #                     properties(class: 'java.util.Arrays$ArrayList') {
+      #                         a(class: 'jenkins.branch.BranchProperty-array') {
+      #                             'jenkins.branch.NoTriggerBranchProperty'()
+      #                         }
+      #                     }
+      #                 }
+      #             }
+      #           }
+      #         }
+      ldap-settings: |
+        jenkins:
+          securityRealm:
+            ldap:
+              configurations:
+                - server: "${LDAP_SERVER}"
+                  rootDN: "${LDAP_ROOT_DN}"
+                  managerDN: "${LDAP_MANAGER_DN}"
+                  managerPasswordSecret: "${LDAP_MANAGER_PASSWORD}"
+                  mailAddressAttributeName: "mail"
+                  userSearch: cn={0}
+                  userSearchBase: "ou=people"
+                  groupSearchBase: "ou=groups"
+              disableMailAddressResolver: false
+              groupIdStrategy: "caseInsensitive"
+              userIdStrategy: "caseInsensitive"
+              cache:
+                size: 100
+                ttl: 300
+      advisor-settings: |
+        jenkins:
+          disabledAdministrativeMonitors:
+            - com.cloudbees.jenkins.plugins.advisor.Reminder
+        advisor:
+          acceptToS: true
+          ccs:
+          - "damien.duportal@gmail.com"
+          email: "jenkins@oblak.com"
+          excludedComponents:
+            - "ItemsContent"
+            - "GCLogs"
+            - "Agents"
+            - "RootCAs"
+            - "SlaveLogs"
+            - "HeapUsageHistogram"
+          nagDisabled: true
+      jenkins-url: |
+        unclassified:
+          location:
+            url: "https://release.ci.jenkins.io"
```

Comment:
Added to please the warning shown in Jenkins UI

[async] To be reported on infra & weekly?